cowprotocol/tenderly-watch-tower
# feat: commander

# Description
Introduces commander to deal with the various program run modes and unifying configuration. 

# Changes

- [x] Add commander
- [x] Template the location for the run logic
- [x] Move types to `types/generated`, relocate `model` and define some interfaces.

## How to test

To review:

Run the CLI and observe the help. Subsequent programs should exit throwing "Not Implemented".

```bash
yarn ts-node ./src/index.ts --help
```

## Related Issues

Closes #35
